yeat/workflows/snakefiles/Long

```diff
@@ -0,0 +1,56 @@
+# -------------------------------------------------------------------------------------------------
+# Copyright (c) 2023, DHS. This file is part of YEAT: http://github.com/bioforensics/yeat
+#
+# This software was prepared for the Department of Homeland Security (DHS) by the Battelle National
+# Biodefense Institute, LLC (BNBI) as part of contract HSHQDC-15-C-00064 to manage and operate the
+# National Biodefense Analysis and Countermeasures Center (NBACC), a Federally Funded Research and
+# Development Center.
+# -------------------------------------------------------------------------------------------------
+
+from shutil import copyfile
+
+
+rule copyinput:
+    output:
+        compressed="seq/input/{sample}/combined-reads.fq.gz"
+    input:
+        reads=lambda wildcards: config["samples"][wildcards.sample]
+    params:
+        combined=lambda wildcards: f"seq/input/{wildcards.sample}/combined-reads.fq"
+    run:
+        for i, inread in enumerate(input):
+            outread = f"seq/input/{wildcards.sample}/reads{i}.fq"
+            if inread.endswith(".gz"):
+                shell("gunzip -c {inread} > {outread}")
+            else:
+                copyfile(inread, outread)
+        shell("cat seq/input/{wildcards.sample}/*.fq > {params.combined}")
+        shell("gzip {params.combined}")
```

Comment:
Here's a change. Before, we copied and zipped up all input fastq files and would pass them into various tools and algorithms as a list of input reads.

For example, 
```
Flye --pacbio-hifi read1.fq.gz read2.fq.gz ...
```

Instead of providing a list of files, I've decided to copy, uncompress, and combine all fastq files into one giant fastq file and then compress it.

I did this because,
1. We can dodge the round about solution of sentinel files `.done`
2. nanoplot can only read in 1 fastq file at a time
3. if there are any tools out there that can only take in 1 read, we can provide the combined fastq file

Now,
```
Flye --pacbio-hifi combined.fq.gz
```

\>\> **This does not apply for paired end input reads.** \<\<